### PR TITLE
Provide information on joining the community

### PR DESCRIPTION
**Pull request by galgeek**
Hi @ashughes1 & @whimboo!

This PR includes a :join command and response for all IRC nicks.

I believe Anthony wants to include some additional functionality for issue #19, so this shouldn't close that issue for now.

### CODE REVIEW

File: bot.js [hunk 1]
```diff
@@ -133,6 +134,14 @@ client.addListener('message', function(from, to, message) {
   if (message.search('[!:]qmo') >= 0) {
     client.say(to, "QMO is short for http://quality.mozilla.org, the official destination for everything related with Mozilla QA");
   }
+
+  if (message.search('[!:]join') >= 0) {
+    client.say(from, "Mozilla QA is a diverse, open community of people pushing " +
+                   "the open web forward by ensuring Mozilla produces the " +
+                   "best technology possible. See https://wiki.mozilla.org/QA " +
+                   "to find out more about getting involved.");
```
Comment (whimboo) [inline bot.js]: nit: Those lines should align with the 'f' of from.

**Comment by whimboo** [comment]
@ashughes1 are you ok with the command named as `:join`?

**Comment by ashughes1** [comment]
> @ashughes1 are you ok with the command named as :join?

Yes, I can't think of a better phrasing for the command.

**Comment by galgeek** [comment]
Hello @whimboo!

Thanks for your feedback. I've updated the PR.

**Comment by galgeek** [comment]
This code has already gotten merged somehow with master.

**Comment by whimboo** [comment]
Looks like I missed to give an update here. Simply have a look at all the commits and you can find that the merge happened with https://github.com/mozilla/testdaybot/commit/d1a0a4e888e7372ffa3476fe4b3d952389ca739c.

**Pull request closed without merging**